dali/ft/filecopy.cpp

```diff
@@ -1736,8 +1736,12 @@ void FileSprayer::locateJsonHeader(IFileIO * io, unsigned headerSize, offset_t &
     Owned<IFileIOStream> in = createIOStream(io);
 
     JsonSplitter jsplitter(srcFormat, *in);
+    if (headerSize > 0)
+        in->seek(headerSize, IFSbegin); // Skip BOM
     headerLength = jsplitter.getHeaderLength();
     footerLength = jsplitter.getFooterLength();
```

Comment:
not new, but doesn't this read the entire json file?  Is there a more efficient way to calculate this.